libraries/plugins/witness/witness_plugin.cpp

```diff
@@ -466,11 +466,13 @@ namespace detail {
 
    void witness_plugin_impl::schedule_production_loop() {
       //Schedule for the next second's tick regardless of chain state
-      // If we would wait less than 50ms, wait for the whole second.
+      // If we would wait less than 50ms, wait for at least another 200ms.
       fc::time_point now = fc::time_point::now();
       int64_t time_to_next_second = 1000000 - (now.time_since_epoch().count() % 1000000);
```

Comment:
Simply change this to `time_to_next_wakeup` and replace `1000000` with `200000`. The values here does be replaced with defined constants at the top of the source file instead of "magic" numbers.